lib/elixir/lib/calendar/iso.ex

```diff
@@ -251,12 +251,26 @@ defmodule Calendar.ISO do
   Converts a `t:System.time_unit/0` to precision.
 
   Integer-based time units always get maximum precision.
+
+  ## Examples
+
+      iex> Calendar.ISO.time_unit_to_precision(:nanosecond)
+      6
+
+      iex> Calendar.ISO.time_unit_to_precision(:second)
+      0
+
+      iex> Calendar.ISO.time_unit_to_precision(1)
+      6
+
   """
+  @doc since: "1.15.0"
```

Comment:
This is technically available since v1.14.3, but we add features in minor (not patch) versions, so maybe this is the better approach?